common/include/pcl/common/impl/bivariate_polynomial.hpp

```diff
@@ -113,11 +115,8 @@ pcl::BivariatePolynomialT<real>::deepCopy (const pcl::BivariatePolynomialT<real>
     gradient_x = new pcl::BivariatePolynomialT<real> ();
     gradient_y = new pcl::BivariatePolynomialT<real> ();
   }
-  real* tmpParameters1 = parameters;
-  const real* tmpParameters2 = other.parameters;
-  unsigned int noOfParameters = getNoOfParameters ();
-  for (unsigned int i=0; i<noOfParameters; i++)
-    *tmpParameters1++ = *tmpParameters2++;
+
+  std::copy_n(other.parameters, getNoOfParameters (), parameters);
```

Comment:
So neat now! :slightly_smiling_face:

Reply:
No issue of size too, int vs size_t doesn't matter 😛